components/stripe/actions/create-payment-intent/create-payment-intent.js

```diff
@@ -0,0 +1,87 @@
+const pick = require("lodash.pick");
+const stripe = require("../../stripe.app.js");
+
+module.exports = {
+  key: "stripe-create-payment-intent",
+  name: "Create a Payment Intent",
+  version: "0.0.1",
+  description: "Create a [payment intent](https://stripe.com/docs/payments/payment-intents)",
+  props: {
+    stripe,
+    amount: {
+      propDefinition: [
+        stripe,
+        "amount",
+      ],
+      required: true,
+    },
+    country: {
+      propDefinition: [
+        stripe,
+        "country",
+      ],
+      required: true,
+      default: "us",
```

Comment:
Does it make sense to incorporate defaults like this into the prop definition itself?